titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/common/replicator/ReplicatedSnapshot.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.runtime.connector.common.replicator;
+
+public abstract class ReplicatedSnapshot {
+
+    public abstract String toSignatureString();
```

Comment:
Can you add docs explaining why this is needed?

Reply:
I have renamed this to `toSummaryString`, and I have added a short comment explaining what is the purpose of this method.